pkg/app/validate_app.go

```diff
@@ -93,13 +95,19 @@ func (v *Validator) Validate(request *v1beta1.AdmissionRequest) (bool, error) {
 		return true, nil
 	}
 
-	if ver.Major() < 3 {
+	// If the app CR does not have the unique version and is < 3.0.0 we skip
+	// the validation logic. This is so the admission controller is not
+	// enabled for existing platform releases.
+	if key.VersionLabel(app) != uniqueAppCRVersion && ver.Major() < 3 {
```

Comment:
Maybe I got it wrong but maybe we should use `or` operator than `and`?

Reply:
We want to skip existing tenant app CRs with version <= 3.0.0. 

But because the unique app CRs are 0.0.0 an `or` won't work.